src/relative-time-element.ts

```diff
@@ -382,6 +383,14 @@ export class RelativeTimeElement extends HTMLElement implements Intl.DateTimeFor
     this.setAttribute('format-style', value)
   }
 
+  get noTitle(): boolean {
+    return this.hasAttribute('no-title')
+  }
+
+  set noTitle(value: boolean | null) {
+    this.setAttribute('no-title', value?.toString() || '')
```

Comment:
This should probably use `toggleAttribute` which will be a bit simpler.

Reply:
I had to pass in a param into the setter even though it's unused... Not sure how else to get around that as I believe it's required 🤔 Maybe it's ok?